resources/META-INF/plugin.xml

```diff
@@ -31,4 +32,16 @@
                 description="Run ruff with fix option">
         </action>
     </actions>
+<!--    <extensions defaultExtensionNs="com.redhat.devtools.lsp4ij">-->
+<!--        <server id="ruffLanguageServer"-->
+<!--                name="Ruff Language Server"-->
+<!--                factoryClass="com.koxudaxi.ruff.lsp.lsp4ij.RuffLanguageServerFactory">-->
+<!--            <description><![CDATA[-->
```

Comment:
Fill the correct description of your ls in html syntax. This description will be used in lsp console